flink-connector-mysql-cdc/src/main/java/com/ververica/cdc/connectors/mysql/debezium/task/MySqlSnapshotSplitReadTask.java

```diff
@@ -77,8 +74,11 @@
     private final EventDispatcher.SnapshotReceiver snapshotReceiver;
     private final SnapshotChangeEventSourceMetrics snapshotChangeEventSourceMetrics;
 
+    private final AbstractMySqlFieldReader mySqlFieldReader;
+
```

Comment:
In line 230, we should use `connectorConfig.getSnapshotFetchSize()` instead of `connectorConfig.getQueryFetchSize()`.